src/main/java/com/cdancy/bitbucket/rest/features/BranchApi.java

```diff
@@ -120,6 +119,31 @@ Branch getDefault(@PathParam("project") String project,
     BranchModel model(@PathParam("project") String project,
                       @PathParam("repo") String repo);
 
+    @Named("branch:get-model-configuration")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/branch-utils/{jclouds.api-version}/projects/{project}/repos/{repo}/branchmodel/configuration")
+    @Fallback(BitbucketFallbacks.BranchModelConfigurationOnError.class)
+    @GET
+    BranchModelConfiguration getModelConfiguration(@PathParam("project") String project,
+                                                   @PathParam("repo") String repo);
+
+    @Named("branch:get-model-configuration")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/branch-utils/{jclouds.api-version}/projects/{project}/repos/{repo}/branchmodel/configuration")
+    @Fallback(BitbucketFallbacks.BranchModelConfigurationOnError.class)
+    @PUT
+    BranchModelConfiguration updateModelConfiguration(@PathParam("project") String project,
+                                          @PathParam("repo") String repo,
+                                          @BinderParam(BindToJsonPayload.class) CreateBranchModelConfiguration config);
+
+    @Named("branch:delete-model-configuration")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/branch-utils/{jclouds.api-version}/projects/{project}/repos/{repo}/branchmodel/configuration")
+    @Fallback(BitbucketFallbacks.FalseOnError.class)
+    @DELETE
+    boolean deleteModelConfiguration(@PathParam("project") String project,
```

Comment:
I'd like to start doing 1 endpoint at most per PullRequest. Makes reviews quicker and easier to do as well as test. Can you split this one up?